src/CocktailInfo/CocktailInfo.js

```diff
@@ -1,20 +1,75 @@
 import { cocktails } from "../mockData";
 import NavBar from "../NavBar/NavBar";
-import { Heading } from "@chakra-ui/react";
+import "./CocktailInfo.css";
+import { Heading, Spinner } from "@chakra-ui/react";
+import { useQuery, gql, useLazyQuery } from "@apollo/client";
+import React, { useEffect, useState } from "react";
+import EditCocktail from "../EditCocktail/EditCocktail";
+import axios from "axios";
+import { from } from "@apollo/client";
+import { RefetchQueriesFunction } from "@apollo/client";
+import { RefetchQueriesResult } from "@apollo/client";
 
+const CocktailInfo = ({ cocktailId }) => {
```

Comment:
Love that with hooks you were able to lighten up the amount of data passed into this component!